docs/install/remote.md

```diff
@@ -50,67 +50,45 @@ Before you begin, ensure that the following prerequisites are met:
       To install all extras, run:<br>
       ```pip install mlrun[complete]```
      
-2. Alternatively, if you already installed a previous version of MLRun, upgrade it by running:
+3. Alternatively, if you already installed a previous version of MLRun, upgrade it by running:
 
     ```sh
     pip install -U mlrun==<version>
     ```
 
-2. Ensure that you have remote access to your MLRun service (i.e., to the service URL on the remote Kubernetes cluster).
+4. Ensure that you have remote access to your MLRun service (i.e., to the service URL on the remote Kubernetes cluster).
 
 ## Configure remote environment
```

Comment:
This list is already in "In this section," and adding this heading pushes all the heading levels down. I suggest either removing it, or renaming it overview and changing the heading level (up one level) of the sections that start with **Using**.
Also there's mismatch in terminology. The heading is client environment, and the list is remote environment. Change the title of the topic?
![image](https://user-images.githubusercontent.com/88145832/215326363-9aebdcc7-255c-4120-bb8b-3a828ebbe207.png)

Reply:
fixed